UPDATING.md

```diff
@@ -6,6 +6,17 @@
 
 # AtomVM Update Instructions
 
+## v0.6.0-beta.0 -> v0.6.0-beta.1
```

Comment:
I'm not sure it belongs to this file, but in case, I would say that is relevant for v0.6.0 -> v0.7.0.

Reply:
It seems important and relevant to anyone with a local clone of the repository right now. But maybe it doesn't belong in this file, but instead prominently posted on the top of the README.md.